src/desktop/components/main_layout/header/index.styl

```diff
@@ -117,6 +103,20 @@
     border-top 1px solid gray-darkest-color
     padding 20px 0px
     avant-garde-size('body')
+.mlh-notification ~ .main-layout-header-user
```

Comment:
If both notifications and user are present the space should be inserted before the notification not before the user element. This selector is to ensure that.

Reply:
#minor. The `mlh-*` naming here for both seems to be inconsistent. It could be already from previous implementation but it's more consistent if the prefix got applied to, say, both containers (or elements).

Reply:
Yea, I haven't named any of those....